### PR TITLE
script to automate installing golang, setting up GOROOT, GOPATH, and GOBIN, and install/test gonomics repo

### PR DESCRIPTION
**Pull request by edotau**
Well tested on MacBooks and lightly tested on linux machines

### CODE REVIEW

File: easyInstall.sh [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    echo 'export GOBIN=${GOPATH}/bin' >> ~/.env
+    echo 'export PATH=$GOPATH/bin:$GOROOT/bin:$PATH' >> ~/.env
+    source ~/.env
+fi
+
+go env -w GOROOT=/usr/local/go
+go env -w GOPATH=$HOME
+go env -w GOBIN=${GOPATH}/bin
+# Checks for the github.com and directory and will create it if it doesn't already exist
+GITHUB=~/src/github.com
```
Comment (DanielSnellings) [inline easyInstall.sh:97]: I think lines 97-117 can be simplified to `go get github.com/vertgenlab/gonomics/...`

**Review by DanielSnellings — APPROVED** [review]
A question and a comment, but otherwise this looks like a great utility!

File: easyInstall.sh [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    source ~/.zshrc
+else
+    echo 'export GOROOT=/usr/local/go' >> ~/.env
+    echo 'export GOPATH=$HOME' >> ~/.env
+    echo 'export GOBIN=${GOPATH}/bin' >> ~/.env
+    echo 'export PATH=$GOPATH/bin:$GOROOT/bin:$PATH' >> ~/.env
+    source ~/.env
+fi
+
+go env -w GOROOT=/usr/local/go
```
[the remark this answers is not in the document]
Comment (edotau) [reply easyInstall.sh:93]: Yeah I'm with you on that. I'll do one more check right now. Yesterday when I running some tests, `go env` was showing the variables as not set

**Review by rimangan — DISMISSED** [review]

**Comment by craiglowe** [comment]
After a discussion at code review I think the consensus was that this is a powerful tool, but probably better suited for the documentation repository.

**Pull request closed without merging**